app/models/ride_mechanic.rb

```diff
@@ -0,0 +1,4 @@
+class RideMechanic < ApplicationRecord
+  belongs_to :ride
+  belongs_to :mechanic
```

Comment:
Looks like you got the db set up eventually, but it sounds like it took up most of your time. This (M2M db relationships) could be a great thing to practice!

Reply:
I will look into that further! Thanks Juliet!